### ISSUE TITLE
Support for DJI 5 SDK

### ISSUE THREAD

**Issue by neilyoung** [DJI-Mobile-SDK-Tutorials/Android-VideoStreamDecodingSample#91]
Is this provided somewhere? Will this app still work with 5.x? Where can I read about the breaking changes w.r.t. video streaming between 4.x and 5.x?

TIA

**Comment by neilyoung** [comment]
Anybody?

**Comment by pktiuk** [comment]
@neilyoung    
Maybe you should contact them via their form: https://djisdksupport.zendesk.com/hc/en-us/requests/new   
Let us know if they will answer

**Comment by neilyoung** [comment]
Thanks. They are responding via the support forum.

**Comment by pktiuk** [comment]
@neilyoung    
Then you could paste link to related forum thread :)

**Comment by neilyoung** [comment]
No, this communication was held using a private support channel by somebody else. I'm not allowed to disclose. Sorry.
